# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../zthread/destdir.patch.cross
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.am.orig	2005-03-13 10:56:18.000000000 +0100
+++ ./Makefile.am	2008-11-12 09:44:49.000000000 +0100
@@ -28,13 +28,13 @@
 ## install the config script
 install-exec-hook:
 	$(mkinstalldirs) $(bindir)
-	$(INSTALL_PROGRAM) $(top_srcdir)/share/zthread-config $(bindir)
+	$(INSTALL_PROGRAM) $(top_srcdir)/share/zthread-config $(DESTDIR)$(bindir)
 
 ## install the config script & m4 macros
 install-data-hook:
 	$(mkinstalldirs) $(datadir)/aclocal
-	$(INSTALL_DATA) $(top_srcdir)/share/zthread.m4 $(datadir)/aclocal
-	$(INSTALL_DATA) $(top_srcdir)/share/pthread.m4 $(datadir)/aclocal
+	$(INSTALL_DATA) $(top_srcdir)/share/zthread.m4 $(DESTDIR)$(datadir)/aclocal
+	$(INSTALL_DATA) $(top_srcdir)/share/pthread.m4 $(DESTDIR)$(datadir)/aclocal
 	mkdir -p $(DESTDIR)$(includedir)/zthread
 	cp -pR $(top_srcdir)/include/zthread $(DESTDIR)$(includedir)/
 
